github/data_source_github_repository.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"net/http"
 	"strings"
 
+	"github.com/google/go-github/v41/github"
```

Comment:
```suggestion
	"github.com/google/go-github/v42/github"
```

Had a recent bump to v42 over in https://github.com/integrations/terraform-provider-github/pull/1035.  This may need an update.

Reply:
I just rebased. 👍